<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QREDataLoaderProperties</class>
 <widget class="QWidget" name="QREDataLoaderProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Ignore lines</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="headerPrefixEdit">
        <property name="toolTip">
         <string>Example: #,//</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Example: #,//</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="linesToSkipEdit">
        <property name="toolTip">
         <string>Example: 1-5, 12</string>
        </property>
        <property name="placeholderText">
         <string>Example: 1-5, 12</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Line numbers:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Starting with:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Data columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Separator:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="separatorCombo">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>&lt;TAB&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;SPACE&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>,</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="spinBox_3">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Read Q</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>from column</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="factorLabel4_8">
          <property name="text">
           <string>multiplied by</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QDoubleSpinBox" name="factorSpinBox4_2">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QComboBox" name="qUnitCombo">
          <item>
           <property name="text">
            <string>1/nm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/Ã…</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QDoubleSpinBox" name="factorSpinBox4_5">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>from column</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="factorLabel4_7">
          <property name="text">
           <string>multiplied by</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QDoubleSpinBox" name="factorSpinBox4_6">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="unitLabel4_2">
          <property name="text">
           <string>given values are in</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Read R</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="factorLabel4_2">
          <property name="text">
           <string>multiplied by</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>from column</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="enableErrorCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Read error</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
